Raven.Client.Lightweight/Connection/Async/AsyncServerClient.cs

```diff
@@ -102,7 +102,8 @@ public OperationCredentials PrimaryCredentials
             this.replicationInformer = replicationInformerGetter(databaseName);
             this.readStripingBase = replicationInformer.GetReadStripingBase(incrementReadStripe);
 
-            this.replicationInformer.UpdateReplicationInformationIfNeeded(this);
+            if (this.url != MultiDatabase.GetRootDatabaseUrl(url))
+                this.replicationInformer.UpdateReplicationInformationIfNeeded(this);
```

Comment:
Yes we do, but we don't need to have a task for the system database. It is very hard and unprobable that anyone will have replication between system databases enabled